code/index.html

```diff
@@ -1,30 +1,182 @@
 <!DOCTYPE html>
-<html>
-  <head>
+<html lang="en">
+
+<head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+    <link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:ital,wght@1,700&family=Fira+Sans&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Southcoast Spa</title>
+</head>
+
+<body>
+    <div class="hero-image">
+        <div class="text-container">
+            <h1>Southcoast Spa</h1>
+            <span>Realxation and wellbeing</span>
+        </div>
+
+        <div class="hamburger">
+            <div></div>
+            <div></div>
+            <div></div>
+        </div>
+        <nav>
+            <a href="#">Find us</a>
+            <a href="#">Products</a>
+            <a href="#">Contact</a>
+        </nav>
+    </div>
+
+
+    <main>
+
+        <!-- Accordion -->
+        <section class="accordion">
+            <h2>Our treatments and activities</h2>
+            <div class="accordion-container">
+                <h3 id="accordion-header-1" class="accordion-header">
+                    <img src="./img/acc-face.jpg" alt="image face">
+                    <span>Face</span>
+                    <span class="plus">+</span>
+                </h3>
+                <div class="accordion-text-container">
+                    <div class="accordion-text">
+                        <ul>
+                            <li>Deep scrub</li>
+                            <li>Relaxing face massage</li>
+                            <li>Mosturising treatments</li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+            <div class="accordion-container">
+                <h3 id="accordion-header-2" class="accordion-header">
+                    <img src="./img/acc-body.jfif" alt="image body">
```

Comment:
Remember that the 'alt' attribute for images should reflect what the image is displaying. Read a bit about it here:
https://www.w3schools.com/tags/att_img_alt.asp